raster/r.his/testsuite/test_r_his.py

```diff
@@ -0,0 +1,90 @@
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class TestRHis(TestCase):
+    hue = "elevation"
+    intensity = "elevation_shaded_50"
+    saturation = "elevation"
+    red = "shadedmap_r"
+    green = "shadedmap_g"
+    blue = "shadedmap_b"
+    bgcolor = "none"
+
+    @classmethod
+    def setUpClass(cls):
+        cls.use_temp_region()
+        cls.runModule("g.region", raster="elevation")
+        cls.elev_shade = "elevation_shaded_relief"
+        cls.use_temp_region()
+        cls.runModule("r.relief", input="elevation", output=cls.elev_shade)
+        cls.runModule(
+            "r.mapcalc", expression=f"{cls.intensity} = {cls.elev_shade} * 1.5"
+        )
+        cls.runModule("r.colors", map=cls.intensity, color="grey255")
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.runModule("g.remove", type="raster", flags="f", pattern="elevation_*")
```

Comment:
Here I would remove the specific names instead of the pattern, just in case someone runs it locally in a PERMANENT mapset of the sample dataset, because that would remove other existing layers.

```suggestion
        cls.runModule("g.remove", type="raster", flags="f", name=[cls.intensity, cls.elev_shade])
```

Reply:
Thank you for explaining it. That again make sense!